Source/EventAnalysis/KTCollinearTrackClustering.hh

```diff
@@ -0,0 +1,127 @@
+/**
+ @file KTCollinearTrackClustering.hh
+ @brief Contains KTCollinearTrackClustering
+ @details Groups collinear tracks into one
+ @author: E. Zayas
+ @date: May 1, 2017
+ */
+
+#ifndef KTCOLLINEARTRACKCLUSTERING_HH_
+#define KTCOLLINEARTRACKCLUSTERING_HH_
+
+#include "KTPrimaryProcessor.hh"
+
+#include "KTSlot.hh"
+#include "KTData.hh"
+#include "KTMemberVariable.hh"
+#include "KTProcessedTrackData.hh"
+
+#include <algorithm>
+#include <set>
+#include <vector>
+#include <list>
+
+
+namespace Katydid
+{
+    
+    /*!
+     @class KTCollinearTrackClustering
+     @author E. Zayas
+
+     @brief Identifies and groups collinear tracks
+
+     @details
+     Clusters tracks together in the slope/intercept plane with an elliptical boundary
+
+     Configuration name: "collinear-track-clustering"
+
+     Available configuration values:
+     - "slope-radius": clustering tolerance in the slope axis
+     - "frequency-radius": clustering tolernace in the intercept axis
+
+     Slots:
+     - "track": void (shared_ptr<KTData>) -- If this is a new acquisition; Adds tracks to the internally-stored set of points; Requires KTProcessedTrackData.
```

Comment:
Write the slot and signal signatures as `void (KTDataPtr)` for clarity, conciseness, and consistency